lib/ansible/modules/cloud/docker/docker_network.py

```diff
@@ -88,6 +88,12 @@
       - absent
       - present
 
+  internal:
+    version_added: 2.5
+    description:
+      - Restrict external access to the network.
+    default: false
```

Comment:
Why not allow a third value, `None` (by omitting default)? That would allow you to not force recreation of existing networks (which already have `internal` set) if you don't specify it?